config/webpack.prod.config.js

```diff
@@ -167,5 +172,8 @@ module.exports = Merge.smart(commonConfig, {
       analyzerMode: 'static',
       openAnalyzer: false,
     }),
+    new webpack.DefinePlugin({
```

Comment:
We use the DotEnv plugin further up to pull environment variables in via `.env` files.  Can we fold this into that?  I'm not sure exactly what happens if both DotEnv and DefinePlugin are used together, perhaps it's fine, but we should keep it standardized.